### PR TITLE
internal: switch to nope.media 11.0.1

### PR DESCRIPTION
**Pull request by mbouron**

### CODE REVIEW

File: libnopegl/meson.build [hunk 1]
```diff
@@ -278,7 +278,7 @@ hosts_cfg = {
 
 lib_deps = [
   cc.find_library('m', required: false),
-  dependency('libnopemd', version: '>= 11.0.0'),
+  dependency('libnopemd', version: '>= 11.0.1'),
```
Comment (ubitux) [inline libnopegl/meson.build:281]: You can drop that we don't have a hard requirement.

File: ngl-tools/meson.build [hunk 2]
```diff
@@ -71,7 +71,7 @@ tool_deps = [
   cc.find_library('m', required: false),
   dependency('libnopegl'),
 ]
-nopemd_dep = dependency('libnopemd', version: '>= 11.0.0')  # for media probing
+nopemd_dep = dependency('libnopemd', version: '>= 11.0.1')  # for media probing
```
Comment (ubitux) [inline ngl-tools/meson.build:74]: Ditto

File: libnopegl/meson.build (hunk 1 above)
Comment (mbouron) [reply libnopegl/meson.build]: Ok.

File: ngl-tools/meson.build (hunk 2 above)
Comment (mbouron) [reply ngl-tools/meson.build]: Ok.

**Review by ubitux — APPROVED** [review]

**Pull request merged**